files/docker/node/addons/entrypoint.sh

```diff
@@ -44,26 +44,28 @@ return 0
 
 export UPDATE_CHECK='N'
 
+[[ -z ${G_BRANCH} ]] && G_BRANCH="master"
```

Comment:
This was intended to be most of the lines in this file, but only shows up as line 47. I suspect you will get the intent.

Reply:
For reference in case this thread gets reviewed in the future **G_BRANCH** logic is handled already. 

* The execution of `guild-deploy.sh` does a `echo "${BRANCH}" > "${CNODE_HOME}"/scripts/.env_branch`. 
  * Later execution will reuse `.env_branch`, unless the branch does not exist in which case it resets to use **master**

Setting of **G_ACCOUNT**:
* Exported in the shell
* Uncomment/set in `env`
* Uncomment/set in `guild-deploy.sh`

Ensures that **cardano-community** is not used. So a forked repository simply sets **G_ACCOUNT** after building from a test/feature branch and as long as `UPDATE_CHECK=Y` it should persist the configurations from the branch originally used and the fork defined in **G_ACCOUNT**  unless the branch is overridden using `guild-deploy.sh -b <BRANCH>`.